src/core/agent/agent.ts

```diff
@@ -227,6 +227,8 @@ class Agent {
       this.signifyClient = new SignifyClient(keriaConnectUrl, bran, Tier.low);
       this.agentServicesProps.signifyClient = this.signifyClient;
       await this.connectSignifyClient();
+      this.connectionService.onConnectionRemoved();
+      this.connectionService.onConnectionAdded();
```

Comment:
These must be at the START of this function. Because connect can fail and then we will be in the offline mode loop and these will never get set